platform/security/sts/security-sts-propertyclaimshandler/src/main/java/org/codice/ddf/security/sts/claims/property/AttributeFileClaimsHandler.java

```diff
@@ -37,57 +44,35 @@
 import org.apache.cxf.sts.claims.ClaimsParameters;
 import org.apache.cxf.sts.claims.ProcessedClaim;
 import org.apache.cxf.sts.claims.ProcessedClaimCollection;
-import org.apache.cxf.sts.token.realm.RealmSupport;
 import org.boon.json.JsonFactory;
 import org.boon.json.ObjectMapper;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class AttributeFileClaimsHandler implements ClaimsHandler, RealmSupport {
+import org.codice.ddf.configuration.SystemBaseUrl;
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(AttributeFileClaimsHandler.class);
+public class AttributeFileClaimsHandler implements ClaimsHandler, SystemHighAttributes {
```

Comment:
❓ Why does this no longer implement `RealmSupport`?

Reply:
The blueprint did not include a service definition for that interface:
https://github.com/codice/ddf/blob/7dd7f72c536717ed4fb479ef336346c9bf47506c/platform/security/sts/security-sts-propertyclaimshandler/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L45-L47

and the override methods `getSupportedRealms()` and `getHandlerRealm()` just returned null. Looks like it was just some leftover dead code from a previous implementation.

Reply:
I am curious about that too